internal/strict/controls/control.go

```diff
@@ -97,6 +100,11 @@ func (ctrl *Control) AddSubcontrols(newCtrls ...strict.Control) {
 	ctrl.Subcontrols = append(ctrl.Subcontrols, newCtrls...)
 }
 
+// SuppressWarning suppresses the warning message from being displayed.
+func (ctrl *Control) SuppressWarning() {
+	ctrl.Suppress = true
+}
+
```

Comment:
It's better not to change the current control, but to return a copy of the control with a new value. Imagine that you need to use the same control, in one place with the warning suppressed, and in another - not.

```suggestion
// SuppressWarning suppresses the warning message from being displayed.
func (ctrl *Control) SuppressWarning() *Control {
        newCtrl := *ctrl
	newCtrl.Suppress = true
	
	return &newCtrl
}

```

don't forget to extend the `Control` interface in `strict/control.go`

```go
type Control interface {
   SuppressWarning() Control
}
```

Reply:
We just mutate enable, so I'll avoid doing this to keep things consistent.